hypha/static_src/src/javascript/apply/determination-template.js

```diff
@@ -20,14 +20,27 @@
         }
 
         getMatchingCopy(value) {
+            const proposal_form = document.querySelector('#id_proposal_form');
```

Comment:
I checked out 3a0b2c290f0aaaeda9ae6f7b7b2cb290a90ba004 and tried `grep -ir id_proposal_form *` hoping to find the template or code that creates an element with `id=id_proposal_form` but I don't see it. Am I missing something or is a change to the template missing? I get the following error on the branch associated with #4142:
```
Uncaught TypeError: proposal_form_field is null
    <anonymous> https://hypha.test/static/js/determination-template.js:53
    EventListener.handleEvent* https://hypha.test/static/js/determination-template.js:51
    <anonymous> https://hypha.test/static/js/determination-template.js:57
```